examples/delegated-access/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @turnkey/delegated-access
 
+## 0.0.8
```

Comment:
I actually don't mind these example changelogs; I think the move might be to explicitly version them though. idk.